.rubocop_todo.yml

```diff
@@ -0,0 +1,21 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2020-09-20 20:08:12 +0100 using RuboCop version 0.76.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: CountComments, ExcludedMethods.
+# ExcludedMethods: refine
+Metrics/BlockLength:
+  Max: 27
+
+# Offense count: 8
+Metrics/CyclomaticComplexity:
+  Max: 14
```

Comment:
Still some offenses to fix.
I think it's better to have a conversation and get feedback about those, though.